src/platforms/aarch64_linux_gnu.rs

```diff
@@ -55,10 +56,12 @@ pub fn build(
         "LLVM building cmake",
     )?;
 
-    crate::utils::command(
-        Command::new("ninja").args(["-C", llvm_build_final.to_string_lossy().as_ref(), "install"]),
-        "LLVM building with ninja",
-    )?;
+    let mut ninja = Command::new("ninja");
```

Comment:
Could you take this opportunity to consolidate the seven duplicates of this code into one location?

Reply:
Good idea, done.